cmd/platform/allocator/command.go

```diff
@@ -18,15 +18,18 @@
 package cmdallocator
 
 import (
+	"fmt"
+
 	"github.com/spf13/cobra"
 
 	cmdallocatormetadata "github.com/elastic/ecctl/cmd/platform/allocator/metadata"
+	cmdutil "github.com/elastic/ecctl/cmd/util"
 )
 
 // Command represents the allocator command
 var Command = &cobra.Command{
 	Use:     "allocator",
-	Short:   "Manages allocators (for ECE installations only)",
+	Short:   fmt.Sprintf("Manages allocators %v", cmdutil.PlatformAdminRequired),
```

Comment:
So we are repeating the same naming pattern a few dozens of times and if - in the future - we will need to change it or improve we will need to do the same which is kinda error prone.
How about introduce a function that will accept the command `message` and wrap it with the permissions message. Something like (ignore the function and variable names - pick those that make more sense)
```go
func shortName(name string) string {
    return fmt.Sprintf("%s %v", name, cmdutil.PlatformAdminRequired)
}
```

WDYT?
This will also remove the `cmdutil` deps from all commands and will eliminate some ugly imports like `sdkcmdutil` 😄

Reply:
I think that makes a lot of sense, Using a function with that 1 param works well.